tests/ops/test_qubit_ops.py

```diff
@@ -498,6 +498,23 @@ def test_CY_decomposition(self, tol):
         decomposed_matrix = np.linalg.multi_dot(mats)
         assert np.allclose(decomposed_matrix, op.matrix, atol=tol, rtol=0)
 
+    def test_CRot_decomposition(self, tol):
+        """Tests that the decomposition of the CRot gate is correct"""
+        phi, theta, omega = [0.5, 0.6, 0.7]
```

Comment:
The other tests above likely don't do it, but it suddenly struck me that we should probably parametrize this test for 2-3 numeric values, _just_ to make sure that the decomposition doesn't only work for 0.5, 0.6, 0.7 😆 

(I doubt that is the case, but just being extra cautious)

Reply:
Always a good idea, and so easy to do